src/binary/decimal.rs

```diff
@@ -20,9 +18,7 @@ const DECIMAL_BUFFER_SIZE: usize = 32;
 const DECIMAL_POSITIVE_ZERO: Decimal = Decimal {
     coefficient: Coefficient {
         sign: Sign::Positive,
-        magnitude: UInt {
-            data: UIntData::U64(0),
-        },
+        magnitude: UInt::ZERO,
```

Comment:
🗺️ I added `ZERO` and `NEGATIVE_ZERO` `const` values to the `Decimal` and `Coefficient` types for easier reference.